cobalt/doc/cvals.md

```diff
@@ -119,6 +119,18 @@ Here are examples of its usage:
     compatibility violations encountered.
 *   **Count.XHR** - The total number of xhr::XMLHttpRequest in existence
     globally.
+*   **Count.MainWebModule.AnimatedImage.Active** - The total number of currently
+    active animated image decoders. Same image from a single URL source rendered
+    multiple times across the content counts as one decoder.
+*   **Count.MainWebModule.AnimatedImage.DecodedFrame** - Total number of decoded
+    frames across all active image decoders. This number resets only when
+    WebModule gets re-created - e.g. page reload, navigation.
+*   **Count.MainWebModule.AnimatedImage.DecodingUnderruns** - Total number of
```

Comment:
What can this data tell us? And how can cert team use this information?

For our use case, if we know each webp is 60 fps. We should just count the DecodedFrame over 1 sec / 60 = # of webp?

So there is nothing such as dropped frames in Webp's world?

Reply:
Starting from the end: Cobalt currently does not drop frames, it stretches/lags the exposure time. E.g frame intended to be displayed for 30 milliseconds can end up 300 milliseconds if CPU was bottlenecked.  See b/290060548

> For our use case, if we know each webp is 60 fps.

If you have 1 webp at fixed 60fps, DecodedFrame should be showing exactly 60, underruns should be 0 and overruns should be 0. If you have 2 of those, DecodedFrames should be 120, and other numbers zero. If CPU is falling behind, you will be getting underruns. If timing in the stack is off, you'll be getting overruns - the latter is less useful for performance checks.

> And how can cert team use this information?

To verify correct playback, inspect the asset you are playing, and verify you are getting all frames decoded in expected timeframe. There should be zero underruns

Reply:
Also updated the language here to say frames, not times.